### PR TITLE
Add Travis integration

### PR DESCRIPTION
**Pull request by amitaibu**
I couldn't really test the PR locally, as most of it is checking Travis installs everything correctly. But it's a start 😄

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "editable",
+  "version": "1.0.0",
+  "description": "Editable represents a value that can be read-only or editable.",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "elm-verify-examples && elm-test"
```
Comment (stoeffel) [inline package.json]: Thanks for your contribution ❤️ .
I would prefer a shell script with this instead of a `package.json`

Comment (amitaibu) [inline package.json]: You mean something like `./execute-test.sh` that will just have `elm-verify-examples && elm-test`?

The `package.json` is indeed quite verbose (could probably be trimmed), only reason for it is just the somewhat standardized `npm install`

**Comment by amitaibu** [comment]
btw, if you could please enable Travis for the repo, so it will pick up this PR

File: package.json (hunk 1 above)
Comment (stoeffel) [inline package.json]: > You mean something like ./execute-test.sh that will just have elm-verify-examples && elm-test?

yes

**Comment by stoeffel** [comment]
enabled travis. 🤘

**Comment by stoeffel** [comment]
merging once travis is happy 💚

**Comment by amitaibu** [comment]
Yeah, just need to fix [this](https://travis-ci.org/stoeffel/editable/jobs/257464106#L1142) elm-format related error

**Comment by amitaibu** [comment]
Last commit should probably be green. PR now also includes Travis badge in the README.

**Comment by amitaibu** [comment]
I've removed `osx` as it takes Travis _really_ long time, and it should probably be enough to get the just linux tested.

**Comment by amitaibu** [comment]
And we're green! 😄

**Pull request merged**